src/Main/Main.js

```diff
@@ -0,0 +1,26 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import Title from '../Title/Title'
+import RestaurantsContainer from '../RestaurantsContainer/RestaurantsContainer'
+
+function Main() {
+  return (
+    <main>
+      <Navbar />
+      <div style={{ backgroundImage: "url('" + burgerImage + "')" }} className="burger-image m-md-3">
+        <div className="d-flex flex-row justify-content-center align-items-center h-100 m-3">
+          <div className="semi-transparent bg-light flex-row justify-content-center align-items-center d-flex m-0 title-box-width p-3">
+            <Title />
+          </div>
+        </div>
+      </div>
```

Comment:
i feel like all these divs outside the title component kind of belong to that component? like i think the code would be more streamlined if all those lines were inside the Title component. do you agree?

Reply:
I think the intention was to simplify the Title component for Chloe and Melody working on S2T1, because they only need to change the contents not the formatting. We'd also consider the containing divs to be more like TitleBox / Banner / etc

Reply:
Making a Banner component would be possible though

Reply:
i think with this explanation it's actually okay as is